health/REFERENCE.md

```diff
@@ -60,6 +60,8 @@ Netdata parses the following lines. Beneath the table is an in-depth explanation
 | [`on`](#alarm-line-on)                              | yes             | The chart this alarm should attach to.                                                |
 | [`os`](#alarm-line-os)                              | no              | Which operating systems to run this chart.                                            |
 | [`hosts`](#alarm-line-hosts)                        | no              | Which hostnames will run this alarm.                                                  |
+| [`plugin`](#alarm-line-plugin)                      | no              | Restrict alarm to only certain plugin.                                                |
```

Comment:
```suggestion
| [`plugin`](#alarm-line-plugin)                      | no              | Restrict a template to only certain plugin.                                                |
```

Reply:
I decided to use `alarm`, instead template, because we are not limiting them to `templates.

Reply:
updated

Reply:
```suggestion
| [`plugin`](#alarm-line-plugin)                      | no              | Restrict an alarm or template to only a certain plugin.                                             |
```